libopae/pluginmgr.c

```diff
@@ -64,6 +64,7 @@ static platform_data platform_data_table[] = {
 };
 
 static int initialized;
+static platforms_detected;
```

Comment:
You're thinking this would allow us to have another shot at platform detection if we failed the first time?